### PR TITLE
Replay requests

### PR DESCRIPTION
**Pull request by bhuvak**
Reads messages from service bus continuously and replays them on a target endpoint.

### CODE REVIEW

File: ReplayRequests/ReplayRequests/Program.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            try
+            {
+                using (var requestsFile = File.CreateText("Rquests.txt"))
+                {
+                    var connectionString = CloudConfigurationManager.GetSetting("Microsoft.ServiceBus.ConnectionString");
+                    var queueClient = QueueClient.CreateFromConnectionString(connectionString, CloudConfigurationManager.GetSetting("QueuePath"));
+                    var cancellationTokenSource = new CancellationTokenSource();
+                    var cancellationToken = cancellationTokenSource.Token;
+                    var task = ContinouslyReadFromQueue(cancellationToken, queueClient, requestsFile);
+                    Console.WriteLine("Press any key to exit...");
```
Comment (yishaigalatzer) [inline ReplayRequests/ReplayRequests/Program.cs]: Is this a pattern we are using for jobs?

File: ReplayRequests/ReplayRequests/Program.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                    Console.ReadKey();
+                    cancellationTokenSource.Cancel();
+                    task.Wait();
+                }
+            }catch(Exception e)
```
Comment (yishaigalatzer) [inline ReplayRequests/ReplayRequests/Program.cs]: What happens if there is an error in one of the requests, does the whole process die?

What's the point of catching the exception and printing it out? The console app already does it by default, and that can be now found through the machine logs. If you catch it, its gone

File: ReplayRequests/ReplayRequests/Program.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            {
+                Console.WriteLine("Exception message : {0} {1} Stack trace : {2}", e.Message, Environment.NewLine, e.StackTrace);
+            }
+        }
+
+        private static async Task ContinouslyReadFromQueue(CancellationToken cancellationToken, QueueClient queueClient, TextWriter requestsFile)
```
Comment (yishaigalatzer) [inline ReplayRequests/ReplayRequests/Program.cs]: This looks like a single threaded, and in the foreground type work. Where you instead probably want to run it on multiple threads to saturate the network (or at least mimic a similar load in production).

Do you plan to run it on its own VM?

File: ReplayRequests/ReplayRequests/Program.cs [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+            if (message != null)
+            {             
+                await ProcessMessage(message,requestsFile);
+            }
+        }
+
+        private static async Task ProcessMessage(BrokeredMessage message, TextWriter requestsFile)
+        {
+            string messageBody = message.GetBody<string>();
```
Comment (yishaigalatzer) [inline ReplayRequests/ReplayRequests/Program.cs]: This area is more for @johnataylor to comment on

**Comment by mishra14** [comment]
Closing this PR as this is stale.

**Pull request closed without merging**